janusgraph-all/pom.xml

```diff
@@ -49,11 +49,6 @@
             <version>${hbase098.core.version}-hadoop2</version>
         </dependency>
         <dependency>
-            <groupId>${project.groupId}</groupId>
-            <artifactId>janusgraph-hadoop</artifactId>
```

Comment:
is it possible this got pulled in by another dependency? `mvn dependency:tree` should tell us exactly what is getting pulled in.